Controller/SuggestionProviderController.php

```diff
@@ -34,6 +35,49 @@ class SuggestionProviderController extends ExceptionController
     protected $suggestionProviders = array();
 
     /**
+     * A set of configuration for the error handling.
+     *
+     * Contains the list of templates defined in the error section of the
+     * cmf_seo configuration.
+     *
+     * Looks like
+     *
+     * array(
+     *  'templates' => array(
```

Comment:
i think i would call it leaking implementation details :-)
but i have no hard opinion on this one, just the feeling that more explicit parameters would be nicer.